ButtPlugin.Core/ButtplugController.cs

```diff
@@ -107,14 +107,17 @@ protected IEnumerator DoStroke(float strokeTimeSecs, int girlIndex)
             int strokeTimeMs = (int)(strokeTimeSecs * 1000) - 10;
             // decrease stroke length gradually as speed approaches the device limit
             double rate = 60f / CoreConfig.MaxStrokesPerMinute.Value / strokeTimeSecs;
-            double margin = rate * rate * 0.3;
+            double min = Mathf.InverseLerp(0, 100, CoreConfig.StrokeZoneMin.Value);

```

Comment:
this shouldn't even work, [LinearCmd ](https://buttplug-spec.docs.buttplug.io/generic.html#linearcmd) expects a value between 0 and 1